jjb/dynamic/scale-ci-pipeline-4.7.yml

```diff
@@ -0,0 +1,337 @@
+- job:
+    block-downstream: false
+    block-upstream: false
+    concurrent: true
+    description: Collection of Jenkins jobs to install, configure, scaleup ocp and
+      run various performance and scale tests on OpenShift across various public and
+      private clouds. <!-- Managed by Jenkins Job Builder -->
+    disabled: false
+    name: scale-ci-pipeline-4.7
```

Comment:
can we have pipeline names that match current names i.e all caps
same for the other pipelines too

Reply:
Fixed it, thanks.

Reply:
You are right about the names @mohit-sheth, there might be something wrong with the xml files, changing the names to use uppercase.

Reply:
Done.